src/views/DataSessionsView.vue

```diff
@@ -14,6 +14,10 @@ const deleteSessionId = ref(-1)
 const showDeleteDialog = ref(false)
 const dataSessionsUrl = store.state.datalabApiBaseUrl + 'datasessions/'
 
+onBeforeMount(()=>{
+	if(!store.getters['userData/authToken']) router.push({ name: 'Registration' })
```

Comment:
Only comment I have is that you could make a getter like `userIsAuthenticated` in the userData portion of the store that checks if the authToken exists. It's mostly syntactic sugar, though